### PR TITLE
chore(publish): add npm scripts

### PR DESCRIPTION
**Pull request by jsomsanith**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
The release process implies : 
* multiple manual commands
* to know some complex commands or to copy/paste them from the wiki release page
* to have some tools installed in global

**What is the new behavior?**
3 npm scripts added : 
* `publish` that run the right lerna command
* `publish-storybook` that build the storybook and trigger surge
* `changelog` that display the git changelog with the right format for release description

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

**Other information**:
Once this PR merged, the wiki (https://github.com/Talend/ui/wiki/Release) needs to be updated

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://172.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://172.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://172.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://172.talend.surge.sh)

File: package.json [hunk 1]
```diff
@@ -16,7 +16,11 @@
     "build-theme": "lerna exec --scope=*theme -- yarn run prepublish && ./copylibs.js --scope=theme",
     "test": "lerna exec --concurrency 1 -- yarn test",
     "start": "lerna exec -- yarn start",
-    "watch": "npm-watch"
+    "watch": "npm-watch",
+    "publish": "lerna publish --force-publish=*",
+    "build-storybook": "export TRAVIS_BRANCH='master' && export TRAVIS_BUILD_DIR=`pwd` && .travis/after_success_static.sh",
+    "publish-storybook": "npm run build-storybook && cd .static && surge",
+    "changelog": "git log --pretty=\"format:%C(bold green)%ad%C(reset) %<(70,trunc)%s\" --date=short --color"
```
Comment (frassinier) [inline package.json]: I must change this line to `git log --pretty=format:\"%C(bold green)%ad%C(reset) %<(70,trunc)%s\" --date=short --color` in order to work

Comment (jsomsanith) [inline package.json]: It works on my mac.
What error do you have ?

File: package.json [hunk 2]
```diff
@@ -16,7 +16,11 @@
     "build-theme": "lerna exec --scope=*theme -- yarn run prepublish && ./copylibs.js --scope=theme",
     "test": "lerna exec --concurrency 1 -- yarn test",
     "start": "lerna exec -- yarn start",
-    "watch": "npm-watch"
+    "watch": "npm-watch",
+    "publish": "lerna publish --force-publish=*",
+    "build-storybook": "export TRAVIS_BRANCH='master' && export TRAVIS_BUILD_DIR=`pwd` && .travis/after_success_static.sh",
+    "publish-storybook": "npm run build-storybook && cd .static && surge",
```
Comment (jmfrancois) [inline package.json]: surge will not work if not installed as global

File: package.json [hunk 3]
```diff
@@ -16,7 +16,11 @@
     "build-theme": "lerna exec --scope=*theme -- yarn run prepublish && ./copylibs.js --scope=theme",
     "test": "lerna exec --concurrency 1 -- yarn test",
     "start": "lerna exec -- yarn start",
-    "watch": "npm-watch"
+    "watch": "npm-watch",
+    "publish": "lerna publish --force-publish=*",
+    "build-storybook": "export TRAVIS_BRANCH='master' && export TRAVIS_BUILD_DIR=`pwd` && .travis/after_success_static.sh",
```
Comment (jmfrancois) [inline package.json]: will not work on windows, so we must specify this in the documentation

File: package.json (hunk 2 above)
Comment (jsomsanith) [inline package.json]: Right --> Installed locally

File: package.json (hunk 3 above)
Comment (jsomsanith) [inline package.json]: Wiki updated : https://github.com/Talend/ui/wiki/Release#warning
But we still have to update it after this PR merged

File: package.json (hunk 1 above)
Comment (frassinier) [inline package.json]: None, my bad.

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://172.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://172.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://172.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://172.talend.surge.sh)

**Pull request merged**